.ci/OpenLDAP_run.sh

```diff
@@ -5,7 +5,7 @@ LDAP_DB=/tmp/ldap_db
 
 echo "Creating database directory"
 
-rm -rf ${LDAP_DB} && mkdir ${LDAP_DB} && cp  /usr/share/doc/slapd/examples/DB_CONFIG ${LDAP_DB}
+rm -rf ${LDAP_DB} && mkdir ${LDAP_DB}
```

Comment:
This DB_CONFIG file doesn't exist at multiple locations and I couldn't find it on my Ubuntu systems as well. New Ubuntu system have the config extracted in the filesystem, maybe the file is obsolete nowadays? 
Besides, the [manpage says](http://manpages.ubuntu.com/manpages/focal/man5/slapd-bdb.5.html) that the file only applies to bdb (Berkley) config backend. But [we have hdb configured](https://github.com/laminas/laminas-ldap/blob/2.13.x/.ci/OpenLDAP/slapd.conf#L31) as backend. So I guess it can be safely removed.